lucene/core/src/java/org/apache/lucene/search/FuzzyTermsEnum.java

```diff
@@ -344,67 +363,4 @@ public BytesRef term() throws IOException {
     return actualEnum.term();
   }
 
-  /**
-   * reuses compiled automata across different segments,
-   * because they are independent of the index
-   * @lucene.internal */
-  public static interface LevenshteinAutomataAttribute extends Attribute {
-    public CompiledAutomaton[] automata();
-    public void setAutomata(CompiledAutomaton[] automata);
-  }
-    
-  /** 
-   * Stores compiled automata as a list (indexed by edit distance)
-   * @lucene.internal */
-  public static final class LevenshteinAutomataAttributeImpl extends AttributeImpl implements LevenshteinAutomataAttribute {
-    private CompiledAutomaton[] automata;
-      
-    @Override
-    public CompiledAutomaton[] automata() {
-      return automata;
-    }
-
-    @Override
-    public void setAutomata(CompiledAutomaton[] automata) {
-      this.automata = automata;
-    }
-
-    @Override
-    public void clear() {
-      automata = null;
-    }
-
-    @Override
-    public int hashCode() {
-      if (automata == null) {
-        return 0;
-      } else {
-        return automata.hashCode();
-      }
-    }
-
-    @Override
-    public boolean equals(Object other) {
-      if (this == other)
-        return true;
-      if (!(other instanceof LevenshteinAutomataAttributeImpl))
-        return false;
-      return Arrays.equals(automata, ((LevenshteinAutomataAttributeImpl) other).automata);
```

Comment:
nit: we have some unused imports after this deletion.

Reply:
fixed